backend/ntest/Integration/IntegrationTests.cs

```diff
@@ -575,6 +575,14 @@ TContent content
             return result;
         }
 
+        protected static string SnapshooterNameHelper(
+            string classname,
+            string testname,
+            string keyname
+        ){
+            return classname + "." + testname + "_" + keyname;
```

Comment:
Please use [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) instead of `+`.